QuizApp/Presentation/Common/UIComponents/PopGradient.swift

```diff
@@ -12,7 +12,10 @@ class PopGradient: CAGradientLayer {
         type = .axial
         startPoint = gradientStartPoint
         endPoint = gradientEndPoint
+    }
 
+    override init(layer: Any) {
+        super.init(layer: layer)
```

Comment:
Why are you overriding it but only calling `super`?

Reply:
On orientation changes view (layer) crashes because it calls this constructor so I had to overridden it :)